assets/js/googlesitekit/data/create-settings-store.test.js

```diff
@@ -52,7 +54,12 @@ describe( 'createSettingsStore store', () => {
 			settingSlugs: [ 'isSkyBlue' ],
 			registry,
 		} );
-		registry.registerStore( storeDefinition.STORE_NAME, storeDefinition );
+
+		registry.registerStore( storeDefinition.STORE_NAME, Data.combineStores(
+			storeDefinition,
+			createErrorStore(),
+		) );
+
```

Comment:
I think this change can be reverted after my above comments are addressed. Potentially we could add an extra Jest test though to ensure that the error store actions are called _if_ the store has the error partial store.

Reply:
Yep, we can just revert it now. Updated.